skyportal/handlers/api/group.py

```diff
@@ -358,3 +365,110 @@ def delete(self, group_id, username):
         self.push_all(action='skyportal/REFRESH_GROUP',
                       payload={'group_id': int(group_id)})
         return self.success()
+
+
+class GroupStreamHandler(BaseHandler):
+    @permissions(['System admin'])
+    def post(self, group_id, *ignored_args):
```

Comment:
Do we need `*ignored_args`?

Reply:
Yes, otherwise you'd get errors like `Error in `/api/groups/8/streams`: post() takes 2 positional arguments but 3 were given` since stream id would be None see endpoint def in `app_server.py` -- `(r'/api/groups(/[0-9]+)/streams(/[0-9]+)?', GroupStreamHandler),`

Reply:
It's similar to `GroupUserHandler` in this respect.